.github/workflows/ci.yml

```diff
@@ -80,19 +80,18 @@ jobs:
       - uses: actions/checkout@v1
       - uses: actions-rs/toolchain@v1
         with:
-          toolchain: stable
-      - name: Build kcov
-        run: |
-          sudo apt-get install libcurl4-openssl-dev libelf-dev libdw-dev cmake gcc binutils-dev libiberty-dev
-          wget https://github.com/SimonKagstrom/kcov/archive/master.tar.gz
-          tar xzf master.tar.gz || true
-          cd kcov-master && mkdir build && cd build && cmake .. && make
-          make install DESTDIR=../../kcov-build && cd ../.. && rm -rf kcov-master && rm -rf master.tar.gz
-      - name: Push test coverage to codecov
-        run: |
-          cargo clean
-          cargo test --no-run
+          toolchain: nightly
+          override: true
+      - uses: actions-rs/cargo@v1
+        with:
+          command: test
```

Comment:
I guess we should remove the `test` action from this file, because we run tests twice.
https://github.com/mgrachev/dotenv-linter/blob/master/.github/workflows/ci.yml#L20